### PR TITLE
docs: update 3.4.0 documents

### PR DESCRIPTION
**Pull request by WoodNeck**

### CODE REVIEW

File: README_KR.md [hunk 1]
```diff
@@ -98,6 +99,37 @@ const flicking = new Flicking("#el", { circular: true });
 var flicking = new eg.Flicking("#el", { circular: true });
 ```
 
+#### DOM 구조에 대해
+⚠️ NOTICE: 저희가 플리킹을 좀 더 쉽고 유연하게 사용하실 수 있도록 노력하고 있지만, 플리킹을 올바르게 초기화하기 위해서 최소한의 구조가 필요합니다.
```
Comment (jongmoon) [inline README_KR.md]: I don't think following phrase should be mentioned.
  - `저희가 플리킹을 좀 더 쉽고 유연하게 사용하실 수 있도록 노력하고 있지만, `

Comment (WoodNeck) [reply README_KR.md]: Okay I'll fix that phrase in both languages.

File: test/unit/events.spec.ts [hunk 2]
```diff
@@ -1,16 +1,15 @@
...
 import { horizontal, vertical } from "./assets/fixture";
-import { createFlicking, createHorizontalElement, cleanup, simulate, waitFor, waitEvent } from "./assets/utils";
+import { createFlicking, createHorizontalElement, cleanup, simulate, waitFor, waitEvent, tick } from "./assets/utils";
 import { merge } from "../../src/utils";
 import Viewport from "../../src/components/Viewport";
 
 declare var sinon: SinonStatic;
 
-describe("Events", () => {
+describe.only("Events", () => {
```
Comment (daybrush) [inline test/unit/events.spec.ts]: @WoodNeck 

Remove `only`.

Comment (WoodNeck) [reply test/unit/events.spec.ts]: Wrong commit (5b311cb) was between commits, I've fixed it.

**Pull request merged**